spec/spec_helper.rb

```diff
@@ -15,6 +15,22 @@
 require 'simplecov'
 SimpleCov.start 'rails'
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+def mock_user
```

Comment:
Nice putting this functionality in the spec helper, this will certainly DRY up the code